en/syntax.tex

```diff
@@ -27,11 +27,12 @@ \subsection{Transitivity} Na'vi marks the subject of transitive and
 is gibberish.}\index{transitivity}
 
 \subsubsection{} Many compound verbs are created by pairing an
-uninflected noun or adjective with the verb stem \N{si} \E{to do, to
-make}, which is only used in these compounds, \N{irayo si} \E{to
-thank}, \N{kavuk si} \E{to betray}.  These verbs are always
-intransitive and use the dative for any object
-(\horenref{syn:case:dative-si}).
+uninflected noun or adjective, or occasionally an interjection,
+with the verb stem \N{si} \E{to do, to make}, which is only used 
+in these compounds, \N{irayo si} \E{to thank}, \N{kavuk si} \E{to betray}.
+Proper names may also be used in very casual conversations as slang.
+These verbs are always intransitive and use the dative for any object
+(\horenref{syn:case:dative-si}
```

Comment:
Seems like a parenthesis and full stop were dropped here

Reply:
Fixed this in latest commit